apps/ledger-live-desktop/src/renderer/components/SyncOnboarding/Manual/SyncOnboardingCompanion.tsx

```diff
@@ -351,14 +361,18 @@ const SyncOnboardingCompanion: React.FC<SyncOnboardingCompanionProps> = ({
     if (deviceOnboardingState?.isOnboarded && !seededDeviceHandled.current) {
       if (deviceOnboardingState?.currentOnboardingStep === DeviceOnboardingStep.Ready) {
         // device was just seeded
-        setStepKey(StepKey.Backup);
+        setStepKey(recoverUpsellRedirection?.enabled ? StepKey.Apps : StepKey.Backup);
         seededDeviceHandled.current = true;
         return;
       } else if (
         deviceOnboardingState?.currentOnboardingStep === DeviceOnboardingStep.WelcomeScreen1
       ) {
         // switch to the apps step
-        __DEV__ ? setStepKey(StepKey.Backup) : setStepKey(StepKey.Apps); // for ease of testing in dev mode without having to reset the device
+        if (recoverUpsellRedirection?.enabled) {
```

Comment:
Nope, this is more clear for reading as we can now it's about the concrete feature flag. This file is large, a lot of definitions here and there, it's better to keep the original name to prevent confusing.